app/pages/lab/workflow-viewer/task.cjsx

```diff
@@ -0,0 +1,313 @@
+React = require 'react'
+Resizable = require 'react-component-resizable'
+{TaskName, RequireBox, HelpButton} = require './task-helpers'
+AnswerList = require './answers'
+Sty = require './jsplumb-style'
+
+# keep track of the max z-index
+max_z = 0
+
+# Firefox needs an offset patch for jsPlumb elements (not sure way)
+isFirefox = typeof InstallTrigger != 'undefined'
+
+# timer to detect when resize stops
+timer = null
+
+# Make a task node
+# Takes `plumbId`, `workflow`, `taskKey`, `pos`, and `taskNumber` as a props
+Task = React.createClass
+  displayName: 'Task'
+
+  getInitialState: ->
+    zIndex: @props.taskNumber + 2
+    task: @props.wf[@props.taskKey]
+    endpoints: []
+    zIndex: @props.taskNumber + 2
+    uuid: 0
+    uuids: []
+    me: null
+    width:
+      min: 0
+      max: 0
+      val: 0
```

Comment:
Nice catch, did not mean to put `zIndex` in two times.  Also thanks for the code, that is exactly what I need for this, `zIndex` needs to be initialized by props, but can be changed when a node is dragged (i.e., the node being dragged is always on top).